### PR TITLE
WIP feat(admin): added settings page to control some look and feel settings

### LINKED ISSUE #8788: Make overriding the favicon easier
Our favicon situation is a little lame, currently. Folks have to override 6+ different files in order to get rid of the Elgg branding. One for every size/type of favicon.

It should be possible to put one SVG or high-quality PNG file somewhere in a plugin and have Elgg generate/override all the different size variations. We could consider allowing folks to upload it from the admin area too, but I figure it'd be best to get a good developer experience down and then we can build any admin-area-upload feature on top of that...

I'm not sure how to make this work with a _single_ file, though. Seems like we can't just use the views system, since we don't have a way to dedupe images. 

Maybe it could be a config option?

``` php
<?php
// config.php
return [
  'favicon' => __DIR__ . "/favicon.svg";
];
```

### PR DESCRIPTION
**Pull request by jdalsem**
Fixes #8788
Ref #10083

What more do we need on this page??

look_and_feel

- [ ] enable/disable rss
- [ ] manifest.json?
- [ ] Upload header image?
- [ ] Upload site logo (header/walledgarden form)

### CODE REVIEW

**Comment by jdalsem** [comment]
Do not take a look at the code yet... just wrote something for the screenshot

**Comment by hypeJunction** [comment]
manifest.json enable/disable (perhaps editor to modify content)

**Comment by hypeJunction** [comment]
Perhaps upload topbar / header logo

**Comment by jdalsem** [comment]
> Perhaps upload topbar

this was is already gone..



> header logo

would that be a replacement for the text? or a full header background image?

**Comment by juho-jaakkola** [comment]
I'm nowadays against adding any new settings that change behavior which is supposed to be duplicated between different environments (dev, staging, production). Settings would ideally be used only for something that is supposed to be different between instances (cache on/off, API secrets, etc).

**Comment by jdalsem** [comment]
> I'm nowadays against adding any new settings that change behavior which is supposed to be duplicated between different environments (dev, staging, production).

that is nearly impossible with plugins that have settings... a better approach would be to allow settings to be exported/imported to be portable

File: languages/en.php [hunk 1]
```diff
@@ -339,7 +340,19 @@
 	'profile:field_too_long' => 'Cannot save your profile information because the "%s" section is too long.',
 	'profile:noaccess' => "You do not have permission to edit this profile.",
 	'profile:invalid_email' => '%s must be a valid email address.',
+		
+	'admin:look_and_feel:info' => 'On this page you can control different items related to the look and feel of the site.',
 
+	'admin:look_and_feel:elgg_branding:label' => 'Remove Elgg branding',
+	'admin:look_and_feel:elgg_branding:help' => 'Throughout the site there are various links and logo\'s that show this site is made using Elgg.',
+	'admin:look_and_feel:elgg_branding:confirm' => 'Are you sure you wish to remove the Elgg branding? Consider donating on https://elgg.org/supporter.php.',
```
Comment (mrclay) [inline languages/en.php]: :)

**Comment by mrclay** [comment]
Still interested in something like this?

**Comment by jdalsem** [comment]
yes

**Comment by hypeJunction** [comment]
Just a thought - you could create a theme assets dir in the data directory and simplecache it via views.php. All the uploads can go in there matching the path of the default view location.

**Comment by jdalsem** [comment]
> Just a thought - you could create a theme assets dir in the data directory and simplecache it via views.php. All the uploads can go in there matching the path of the default view location.

good thought... you should join the core developers with all you good ideas :)

**Comment by hypeJunction** [comment]
Lol. Been there, done that. I will just offer good thoughts and appreciation

**Comment by jdalsem** [comment]
Partially replaced by #11430

**Pull request closed without merging**